File: Java/Array-1/sum2/Sum2.java
package sum2;

public class Sum2 {
    //Retorne a soma dos 2 primeiros elementos de um Array, se ele tiver menos que isso, retorne o que ouver e caso seu
    //"length" seja 0, retorne 0.

    public int sum2(int[] nums) {
        int sum = 0, tam = 2;
        if (nums.length < 2) tam = nums.length;
        for (int i = 0; i < tam; i++) {
            sum += nums[i];
        }
        return sum;
    }
}

File: Python/List-1/sum2.py
"""
Dada uma lista numérica "nums", retorne a soma dos seus 2 primeiros elementos, ou que
quer que ela tenha até lá. Retorne 0 caso tenha tamanho 0.
"""


def sum2(nums):
    return sum(nums[:2])


print(sum2([1, 2, 2]))
